packages/block-editor/src/components/block-variation-transforms/index.js

```diff
@@ -95,6 +98,49 @@ function VariationsDropdown( {
 	);
 }
 
+function VariationsToggleGroupControl( {
+	className,
+	onSelectVariation,
+	selectedValue,
+	variations,
+} ) {
+	return (
+		<fieldset className={ className }>
+			<Flex>
+				<ToggleGroupControl
+					label={ __( 'Transform to variation' ) }
+					value={ selectedValue }
+					isBlock
```

Comment:
Regarding @jasmussen point about the border: The `isBlock` prop is what controls the border visibility for the moment. It looks like the `Flex` component is being used here to undo what the `isBlock` prop is for anyway, so I think all we need to do is remove both the Flex and `isBlock` prop 🙂